examples/00-mapdl-examples/BodyFlux-Averaging.py

```diff
@@ -0,0 +1,324 @@
+"""
+.. _ref_solenoid_magnetostatic_2d:
+
+=======================================
+Analysis of a 2D Magnetostatic Solenoid
+=======================================
+
+This example shows how to gather and plot results with material discontinuities
+across elements (Power graphics style) versus the default full average results
+(Power graphics style).
+
+
+Description
+===========
+
+Mechanical APDL has two averaging methods depending the method used for
+presenting results:
+
+* **Full graphics** presents the entire selected model with node averaged results.
+  In the case of an node shared by two or more elements that have differing
+  materials, the stress field is discontinuous across the element boundary
```

Comment:
Full graphics has continuous results across element material boundaries.

Reply:
"materials, the stress field is continuous across the element material boundary"